macros/edr/materializations/tests/failed_test_result_count.sql

```diff
@@ -0,0 +1,34 @@
+{% macro get_failed_test_result_count(flattened_test) %}
```

Comment:
The naming seems a bit inconsistent.
Do you think we should maybe call it `failed_count` everywhere?

Reply:
changed it to `failed_row_count` everywhere (as per your next comment)